public_html/profiles/social/themes/socialbase/gulpfile.js

```diff
@@ -164,7 +164,9 @@ gulp.task('script-components', function() {
   return gulp.src([
       options.theme.js + "components/waves.js",
       options.theme.js + "components/offcanvas.js",
-      options.theme.js + "components/forms.js"
+      options.theme.js + "components/forms.js",
+      options.theme.js + "components/tablesaw.js",
+      options.theme.js + "components/tablesaw-init.js"
```

Comment:
@bogdan-hepting can you make tablesaw a separate library in the theme? you can file the sass in under the components folder just like pickadate and the js in the libs folder. Then compile it to a loose css and js file which you include